src/FINDOLOGIC/Export/CSV/CSVItem.php

```diff
@@ -129,7 +129,7 @@ private function sanitize($input, $stripTags = true)
             $input = strip_tags($input);
         }
 
-        $sanitized = preg_replace('/\t/', ' ', $input);
+        $sanitized = preg_replace('/[\t\n]/', ' ', $input);
```

Comment:
Given that we already strip tags, and also replace tabs with spaces, my concerns from our previous verbal discussions have been dispelled. :white_check_mark:

Reply:
:+1: